### DISCUSSION TITLE
New user for the templae - why my view is different from the youtube video?

### DISCUSSION THREAD

**Discussion by gtclipsy** [BryanJenksCommunity/FAQ#188] [Q&A] [answered]
I'm really new to obsidian so sorry about this silly question! I opened your template vault on my computer, but I dont know why my appearance is different from yours (the left side bar). I was trying to integrate paper from Zotero to Obsidian by following your youtube video, but I cant even find the open command palette. Any suggestion will be super appreciated! 
![image](https://github.com/BryanJenksCommunity/FAQ/assets/172202965/72a02eea-5d74-4c34-8804-de9dd782483a)

**Comment by tallguyjenks** [comment, marked as answer]
command palette is linked to hotkey, `⌘⇧P` or ctrl+shift+P on windows
the folder colors are not there because obsidian updated the app structure and the CSS needs to be updated is all
